backend/graphql/resolvers/lessonResolvers.ts

```diff
@@ -1,10 +1,12 @@
+import { GraphQLScalarType, Kind } from "graphql";
```

Comment:
Please resolve these linter errors. Some things it fixes automatically (like formatting), but things like unused imports it doesnt.